src/app/sitemap.ts

```diff
@@ -0,0 +1,25 @@
+import { CATEGORIES, SITE_URL } from '@/constants';
+import { MetadataRoute } from 'next';
+
+export default function sitemap(): MetadataRoute.Sitemap {
+    const staticPages = [
```

Comment:
Can you restructure this to make it more DRY, please?

```
const staticPaths = [
    ...
]

const dynamicPaths = [
    ...CATEGORIES.map(...)
]

const allPaths = [...staticPaths, ...dynamicPaths];

const sitemap = [
    allPaths.map((path) => ({
        url: `${SITE_URL}/path`,
        lastModified: new Date(),
    })
]

return sitemap;
```

Reply:
Ok, done